prowler/providers/aws/services/backup/backup_service.py

```diff
@@ -13,6 +13,9 @@ class Backup(AWSService):
     def __init__(self, audit_info):
         # Call AWSService's __init__
         super().__init__(__class__.__name__, audit_info)
+        self.backup_plan_arn_template = f"arn:{self.audited_partition}:backup:{self.region}:{self.audited_account}:backup-plan"
```

Comment:
`self.region` or `client.region` since we want to include the resource region here.

Reply:
This applies to all the services that has an ARN template.